### ISSUE TITLE
Parent click

### ISSUE THREAD

**Issue by kaotisk-hund** [fraction/oasis#16]
**What version of this package are you using?**
2.5.3

**What operating system, Node.js, and npm version?**
Fedora 30,v10.16.3 and 6.10.2

**What happened?**
```
TypeError: isNestedReply is not a function
    at Promise (/home/kaotisk/.local/share/yarn/global/node_modules/@fraction/oasis/src/pages/models/post.js:334:13)
    at new Promise (<anonymous>)
    at getRootAncestor (/home/kaotisk/.local/share/yarn/global/node_modules/@fraction/oasis/src/pages/models/post.js:317:38)
    at Object.fromThread (/home/kaotisk/.local/share/yarn/global/node_modules/@fraction/oasis/src/pages/models/post.js:456:32)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

**What did you expect to happen?**
Open the parent of a post

**Are you willing to submit a pull request to fix this bug?**
Not sure I can help... I understand that somewhere, ( possibly at @fraction/oasis/src/pages/models/post.js:334:13 ) a function is expected but a promise is returned. Maybe it's even more complicated or simple but not the case I guess.

**Comment by christianbundy** [comment]
Hey, thanks for the bug report and sorry to hear you're having a hard time.

I'll look into the code to see if I can figure out what's going on, but if you have time could you double-check whether this happens when you select *parent* for any message or just some in particular?

Thanks again!

**Comment by kaotisk-hund** [comment]
Yes :S It does it in all random parents I requested to check this out.

**Comment by kaotisk-hund** [comment]
Okay, it seems to doing the same thing when I click "link".

**Comment by christianbundy** [comment]
Thanks, that's helpful! I'll keep looking into this and let you know when I've got some more info (or a fix!).

**Comment by christianbundy** [comment]
Looks like a friendly rascal [removed that API](https://github.com/ssbc/ssb-thread-schema/commit/b0ce3a6b984fd7f9c902d24360d25b6bf66889b7) and published as a semver patch. :scream: 

I'll fix this immediately.

**Comment by christianbundy** [comment]
Fixed in 2.6.2!
